### PR TITLE
Make status card look more like träwelling

### PR DESCRIPTION
**Pull request by DRSchlaubi**
image

This makes the status card look almost identical to träwelling.org with the exception being the "next station" field which is currently disabled because of missing API data

### CODE REVIEW

File: app/src/main/res/layout/card_checkin_overview.xml [hunk 1]
```diff
@@ -95,249 +84,288 @@
...
+                        app:layout_constraintLeft_toLeftOf="parent"
+                        app:layout_constraintRight_toRightOf="parent"
+                        app:layout_constraintTop_toBottomOf="@id/icon_product" />
+
+                    <TextView
+                        android:id="@+id/check_in_body"
+                        style="@style/Theme.Traewelling.PrimaryColorTexts"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="4dp"
```
Comment (jheubuch) [inline app/src/main/res/layout/card_checkin_overview.xml:242]: Can we have a little more margin on this? I'd say about 8dp? Imo it looks a bit squeezed at the moment

**Review by jheubuch — CHANGES_REQUESTED** [review]

**Comment by jheubuch** [comment]
Looks very good, thank you! On your screenshot the destination station "Marcel-Breuer-Ring, Erfurt" is cut off on the baseline. Can we have this fixed please? Then I would be fine to merge 🤗

**Comment by DRSchlaubi** [comment]
Will check how I can fix this asap

**Review by jheubuch — APPROVED** [review]

**Comment by jheubuch** [comment]
I had a look at this and also displayed a status of mine with a long text without line breaks.
It came that the text isn't wrapped at the moment:
image

See status: https://traewelling.de/status/536850

**Comment by DRSchlaubi** [comment]
For some reason I can't see that check-in, but I will try to fix it

**Comment by jheubuch** [comment]
Now for really long status bodies the following happens...
image

https://traewelling.de/status/534555

**Comment by DRSchlaubi** [comment]
I noticed that, but frankly I barely know enough about Android layouting to get this working

**Comment by jheubuch** [comment]
I see, I'll have a look at it myself :)

**Comment by jheubuch** [comment]
Now I got it like that, I think, that's fine:
image

**Comment by DRSchlaubi** [comment]
> Now I got it like that, I think, that's fine: image

Looks good :+1:

**Pull request merged**